src/sth_helper.js

```diff
@@ -0,0 +1,116 @@
+/* globals module, process */
+
+(function() {
+  "use strict";
+
+  var sthConfig, sthLogger;
+
+  function stopServer(server, callback) {
```

Comment:
JSDoc comments included! ;) Sorry about that, I forgot to do it from the beginning ;) I'll mention you as soon as I pushed the changes to the repo ;) Thanks!